### ISSUE TITLE
Header. Focus indicator is missing

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#47]

**Comment by sastels** [comment]
On the English page the signature isn't a link (ie not clickable) and so focus skips over it. Perhaps we make the French page the same?
